provider/resource_keycloak_users_permissions.go

```diff
@@ -0,0 +1,272 @@
+package provider
+
+import (
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+)
+
+func resourceKeycloakUsersPermissions() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKeycloakUsersPermissionsCreate,
+		Read:   resourceKeycloakUsersPermissionsRead,
+		Delete: resourceKeycloakUsersPermissionsDelete,
+		Update: resourceKeycloakUsersPermissionsUpdate,
+		Importer: &schema.ResourceImporter{
+			State: resourceKeycloakUsersPermissionsImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"realm_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Required: true,
+				ForceNew: true,
```

Comment:
Did you mean to mark this as `ForceNew`?  This means that an update to the `enabled` attribute will actually cause the resource to be deleted and re-created.

Reply:
if I understand how it should be:
I update from "ForceNew + Required" to "Computed" then user permissions are enabled when "keycloak_users_permissions" is declared and deleted if "keycloak_users_permissions" is not present.

Reply:
Yeah, I think it's okay for the existence of this resource to imply that this feature is enabled.  That makes sense to me.

Although, the `enabled` attribute is kinda useless now.  It's always going to be `true`, right?  The only case where it would be `false` is if the resource was deleted, in which case, you wouldn't have access to the attribute anyways.

It might be best to just remove this attribute entirely.